sys/dev/ath/ath_hal/ah_internal.h

```diff
@@ -639,6 +639,7 @@ extern	void *ath_hal_malloc(size_t);
 extern	void ath_hal_free(void *);
 
 /* common debugging interfaces */
+#define AH_DEBUG
```

Comment:
Probably AH_DEBUG shouldn't be enabled on our main branch

Reply:
Done